tests/utils/esxcli/cmd.go

```diff
@@ -47,3 +56,55 @@ func KillVM(esxHostIP, worldID string) bool {
 	}
 	return true
 }
+
+// RetrieveVMNameFromIP util retrieves VM  name from passed VM IP
+//esxcli vm.info -vm.ip=10.20.104.62 -json | jq -r .VirtualMachines[].Name
+func RetrieveVMNameFromIP(ip string) string {
+	log.Printf("Finding VM name from IP Address [%s]\n", ip)
+	cmd := esxcli.VMInfoByIP + ip + esxcli.JSONTypeOutput + "| " + esxcli.JSONParser + esxcli.VMName
```

Comment:
Can "esxcli.JSONTypeOutput + vmName + " | " + esxcli.JSONParser" be a constant in this file and use it here and in other functions.